homework-1_js/script.js

```diff
@@ -0,0 +1,62 @@
+// 1 Запросите у пользователя его имя и выведите в ответ: «Привет, его имя!».
+let userName = prompt("Как тебя зовут ?");
```

Comment:
тут можна `const`, якщо не далі в коді не переприсвоюєш
```suggestion
const userName = prompt("Как тебя зовут ?");
```